components/scripts/lib/build-scan-parse.sh

```diff
@@ -152,9 +152,9 @@ calculate_pending_savings() {
   fi
 }
 
-# The _pending_ build time is an estimation of the build time if all cacheable
-# tasks had been avoided.
-calculate_pending_build_time() {
+# The _pending_ savings build time is an estimation of the build time if all
```

Comment:
Change to `pending_savings_build_time`

Reply:
Done in 044a7e5d6b270ffa4cc0175933bfc3f6b0cbeece